src/02-common-principles.md

```diff
@@ -128,7 +128,7 @@ participant can exist only at participant level directory, i.e
 specific to a participant is to be declared only at top level of dataset for eg:
 `task-sist_bold.json` must be placed under `/dataset/task-sist_bold.json`
 
-Example 1: Two JSON files at same level that are applicable for NIfTI file.
+Example 1: Two JSON files that are erroneously at the same level.
```

Comment:
looking at example -- it is even more screwed up I guess (on purpose or not) -- it is `sub-01` but example .json files on top are for `sub-test`...  
text then doesn't make it clearer and also talks about `task-overtverbgeneration_run-2_bold.json` which is not there exactly (I feel that `sub-test_` was appended incorrectly at some point without due care)

Reply:
I agree that the naming error should not be part of it, the demonstration of the error should be on the inheritance only. I have corrected it.